lib/puppet/parser/functions/hash2plist.rb

```diff
@@ -0,0 +1,26 @@
+# 
+# hash2plist.rb
+#
+
+require 'puppet/util/plist'
+
+module Puppet::Parser::Functions
+  newfunction(:hash2plist, :type => :rvalue , :doc => <<-EOS
+Takes a hash as an input and returns a plist as a string useful
```

Comment:
`as a string useful` ⇒ `as a string. Useful`

however: an api documentation shouldn't have to justify a function's existence.
it should do so, by explaining what it does.